### DISCUSSION TITLE
ERROR: failed to send payment: Sending(RouteNotFound)

### DISCUSSION THREAD

**Discussion by ConorOkus** [lightningdevkit/rust-lightning#2728] [Help] [answered]
I'm using the ldk-sample to open a channel to a peer running locally on a mobile simulator in regtest.

Currently unable to send payments due to no route being found. Also the channel opens but the `is_channel_ready` flag remains false. 

Also seeing `Searching for a route from payer 020dc0dd6ffd95b630e3df99bef3cc94873292cf891b3b409569b25e2215ddbeb2 to payee node id 0297e215bbd6a736e2e57b338876081ed3a4dc1515e445607fd7886ffa1227b9f2 with MPP and 0 first hops overriding the network graph with a fee limit of 50100 msat`

Logs - [logs.txt](https://github.com/lightningdevkit/rust-lightning/files/13322879/logs.txt)

**Comment by tnull** [comment]
> Currently unable to send payments due to no route being found. Also the channel opens but the `is_channel_ready` flag remains false.

How are you syncing the mobile node? This would indicate that the funding transaction is never marked confirmed with the node.

**Reply by ConorOkus** [reply]
https://github.com/ConorOkus/uMlando-wallet/pull/41/files#diff-1adcddad549fde3fd5220682e24cf9e4b7b02dee80e1c78a52a39dcada1a9ddd

**Reply by ConorOkus** [reply]
The same is also true when creating outbound channels and trying to send payments

**Reply by TheBlueMatt** [reply]
Can you add some instrumentation/logging there and see if/when its connecting the funding transaction?

**Reply by ConorOkus** [reply]
Okay back to this now with some more logs from the mobila app [logs.txt](https://github.com/lightningdevkit/rust-lightning/files/13398045/logs.txt)

**Reply by ConorOkus** [reply]
And logs from the ldk-sample node [logs.txt](https://github.com/lightningdevkit/rust-lightning/files/13398066/logs.txt)

**Reply by tnull** [reply, marked as answer]
Mh, when looking at the mobile logs it doesn't seem like you are connecting any blocks after the channel open. So the sample node sends `ChannelReady`, but the mobile node doesn't as it never sees the funding tx confirmed. 

Will follow-up on the syncing code on Discord.
